fundingmanager.go

```diff
@@ -1283,8 +1403,14 @@ func (f *fundingManager) waitForFundingWithTimeout(completeChan *channeldb.OpenC
 			// The fundingManager is shutting down, will resume
 			// waiting for the funding transaction on startup.
 			return
-		case <-waitingDoneChan:
-			close(doneChan)
+		case shortChanID, ok := <-waitingConfChan:
+			if !ok {
+				fndgLog.Errorf("waiting for funding confirmation" +
```

Comment:
This is getting logged twice, both here and in Start. I would drop this log line and just close & return here.